core/src/test/java/com/taobao/arthas/core/util/StringUtilsTest.java

```diff
@@ -1,12 +1,22 @@
 package com.taobao.arthas.core.util;
 
+import java.util.Properties;
+import java.util.TreeSet;
 import org.junit.Assert;
+import org.junit.Rule;
 import org.junit.Test;
 
+import org.junit.rules.ExpectedException;
+import java.util.ArrayList;
```

Comment:
The import statements could be grouped more neatly - eg. all java.util items together